utils/src/utils/openUrl.ts

```diff
@@ -3,11 +3,9 @@
  *  Licensed under the MIT License. See License.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import open = require("open");
+import * as vscode from 'vscode';
 
 export async function openUrl(url: string): Promise<void> {
     // Using this functionality is blocked by https://github.com/Microsoft/vscode/issues/85930
-    // await vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(url));
-
-    await open(url);
+    await vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(url));
```

Comment:
I'm sorry I think this has been discussed already, but why can't we use `vscode.env.openExternal` here?

Reply:
VSCode is over-encoding some parts of a URI: https://github.com/Microsoft/vscode/issues/85930

However, the workaround suggested--pass a `string` and use `as unknown as vscode.Uri` to placate Typescript--should work. The code definitely very deliberately supports strings being passed in, even if the type declaration doesn't say so.

I think we should change this `openUrl` utility method to accept either `string` or `vscode.Uri`, and pass unchanged.

Reply:
I made this change in resources, but I think that I'll just do it here and delete it everywhere else?